build.sbt

```diff
@@ -20,7 +20,7 @@ lazy val scalaCheckVersion = "1.15.4"
 lazy val munit = "0.7.29"
 lazy val munitDiscipline = "1.0.9"
 
-lazy val algebraVersion = "2.2.3"
+lazy val algebraVersion = "2.7.0"
```

Comment:
Should this be rename to `catsVersion`?

Reply:
Yeah, `catsVersion` = `algebraVersion` now. But since we don't explicitly depend on cats, I think this is fine for now.